### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by striezel**
Fix some typos, no functional changes intended.

### CODE REVIEW

**Comment by cclauss** [comment]
Please rebase

**Review by benmccann — APPROVED** [review]
lgtm once merge conflicts are resolved

**Comment by striezel** [comment]
> Please rebase

@cclauss Done.

**Review by cclauss — APPROVED** [review]
Thank much!

**Pull request merged**
